NuGetPackageVerifier.json

```diff
@@ -6,13 +6,20 @@
         "packages": {
             "Microsoft.VisualStudio.Web.CodeGeneration": { },
             "Microsoft.VisualStudio.Web.CodeGeneration.Core": { },
+            "Microsoft.VisualStudio.Web.CodeGeneration.Design": { },
             "Microsoft.VisualStudio.Web.CodeGeneration.EntityFrameworkCore": { },
             "Microsoft.VisualStudio.Web.CodeGeneration.Templating": { },
             "Microsoft.VisualStudio.Web.CodeGeneration.Tools": { },
             "Microsoft.VisualStudio.Web.CodeGeneration.Utils": { },
             "Microsoft.VisualStudio.Web.CodeGenerators.Mvc": { }
         }
     },
+    "adx-nonshipping": {
+        "rules": [],
+        "packages": {
+            "Microsoft.VisualStudio.Web.CodeGeneration.Sources": { }
```

Comment:
In that case, I recommend not publishing as a package. It's easier to use a shared source folder, something like...
```json
"buildOptions": {
     "compile": {
         "include": "../Shared/*.cs"
    }
}
```